Makefile

```diff
@@ -70,6 +77,37 @@ docker_build_all_sandboxes: docker_build_node_sandbox docker_build_python_sandbo
 .PHONY: docker_build_all
 docker_build_all: docker_build_all_sandboxes docker_build_analysis_image docker_build_scheduler_image
 
+#
+# Builds then pushes analysis and sandbox images
+#
+
+docker_push_%:
+	docker push --all-tags ${REGISTRY}/$(IMAGE_NAME)
+
+docker_push_analysis_image: docker_build_analysis_image
+docker_push_scheduler_image: docker_build_scheduler_image
+docker_push_node_sandbox: docker_build_node_sandbox
+docker_push_python_sandbox: docker_build_python_sandbox
+docker_push_ruby_sandbox: docker_build_ruby_sandbox
+docker_push_packagist_sandbox: docker_build_packagist_sandbox
+docker_push_crates_sandbox: docker_build_crates_sandbox
+docker_push_static_analysis_sandbox: docker_build_static_analysis_sandbox
+
+docker_push_analysis_image: IMAGE_NAME=analysis
+docker_push_scheduler_image: IMAGE_NAME=scheduler
+docker_push_node_sandbox: IMAGE_NAME=node
+docker_push_python_sandbox: IMAGE_NAME=python
+docker_push_ruby_sandbox: IMAGE_NAME=ruby
+docker_push_packagist_sandbox:	IMAGE_NAME=packagist
+docker_push_crates_sandbox: IMAGE_NAME=crates.io
+docker_push_static_analysis_sandbox: IMAGE_NAME=static-analysis
```

Comment:
Might be worth grouping these together like the build versions above.

Reply:
Done